docs/source/usage.rst

```diff
@@ -444,6 +446,8 @@ corrupt data::
 Object history
 --------------
 
+.. versionadded:: 0.4.0
```

Comment:
why did you add `versionadded` for these old sections?

Reply:
Thanks for catching 
I added the missing versionadded tags but missed updating the version values.

removed that code!